OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -159,7 +159,7 @@ public interface INotifyOtherProduction { void UnitProducedByOther(Actor self, A
 	public interface INotifyDelivery { void IncomingDelivery(Actor self); void Delivered(Actor self); }
 
 	[RequireExplicitImplementation]
-	public interface INotifyMineLaying { void MineLaying(Actor self, CPos location); void MineLaid(Actor self, Actor mine); }
+	public interface INotifyMineLaying { void MineLaying(Actor self, CPos location); void MineLaid(Actor self, Actor mine); void MineLayingCanceled(Actor self, CPos location); }
```

Comment:
at this point it would be better to expand the bracket and define the interface across multiple lines